src/core/orderbook.js

```diff
@@ -842,7 +842,7 @@ OrderBook.prototype.updateOwnerOffersFundedAmount = function(account) {
 OrderBook.prototype.notify = function(transaction) {
   const self = this;
 
-  if (!this._subscribed) {
+  if (!(this._subscribed && this._synced)) {
```

Comment:
I was just looking at this issue too. It was possible for a `notify` to create a `model` event before it gets synchronized.